pyproject.toml

```diff
@@ -30,21 +160,57 @@ exclude = '''
 )
 '''
 
+[flake8]
+ignore =[
+  "E203",
+  "E501",
+  "W503",
+]
+# docstring-convention = ""
+max-line-length = 88
+max-complexity = 18
+select = [
+  "B",
+  "C",
+  "E",
+  "F",
+  "W",
+  "T4",
+  "B9"
+]
+
+[tool.ruff]
+exclude = [
+    ".bzr",
+    ".direnv",
+    ".eggs",
+    ".git",
+    ".hg",
+    ".mypy_cache",
+    ".nox",
+    ".pants.d",
+    ".pytype",
+    ".ruff_cache",
+    ".svn",
+    ".tox",
+    ".venv",
+    "__pypackages__",
+    "_build",
+    "buck-out",
+    "build",
+    "dist",
+    "node_modules",
+    "venv",
+]
+# per-file-ignores = []
+line-length = 88
+target-version = "py37"
+# Allow autofix for all enabled rules (when `--fix`) is provided.
+fixable = ["A", "B", "C", "D", "E", "F", "R", "S", "W", "U"]
+
 [tool.mypy]
 ignore_missing_imports = true
 exclude = [
     "docs/*",
     "build/*"
-    ]
-
-[tool.coverage.run]
-omit = [
-    "clarity/_version.py"
 ]
-
-
-[tool.ruff]
-line-length = 88
-# pyflakes, pycodestyle, isort
-select = ["F", "E", "W", "B", "I001"]
```

Comment:
Have these ruff selections gone? I think by default ruff only applies F and E but I found W (codestyle-warnings) , B (flake8-bugbear) and I001 (unsorted-imports) useful too.

Reply:
Sorry, missed copying those over (twice, once when doing it and once when checking). Reinstated.